cdi-core/src/main/java/com/linkedin/cdi/connection/S3Connection.java

```diff
@@ -59,6 +54,7 @@
  */
 public class S3Connection extends MultistageConnection {
   private static final Logger LOG = LoggerFactory.getLogger(S3Connection.class);
+  private static final String UPLOAD_S3_KEY = "s3key";
```

Comment:
Please add some documentation here https://github.com/linkedin/data-integration-library/blob/2b4b5f15b0a17affce4912732d939346c8cb38d9/docs/parameters/ms.secondary.input.md?plain=1#L55 to explain the usage of this new parameter